src/main/resources/org/rust/ide/colors/highlighterDemoText.rs

```diff
@@ -12,6 +12,8 @@ pub enum <ENUM>Flag</ENUM> {
     <ENUM_VARIANT>Ugly</ENUM_VARIANT>
 }
 
+const <CONSTANT>QUALITY</CONSTANT>: <ENUM>Flag</ENUM> = <ENUM>Flag</ENUM>::<ENUM_VARIANT>Good</ENUM_VARIANT>;
```

Comment:
Let's also add examples for `MUT_PARAMETER` and `KEYWORD_UNSAFE`
We already have unsafe function so it's enough to add `<KEYWORD_UNSAFE>` tag